Swift/BRCryptoDemo/CoreDemoAppDelegate.swift

```diff
@@ -135,7 +143,7 @@ class CoreDemoAppDelegate: UIResponder, UIApplicationDelegate, UISplitViewContro
     func applicationWillEnterForeground(_ application: UIApplication) {
         // Called as part of the transition from the background to the active state; here you can
         // undo many of the changes made on entering the background.
-        system.start(networksNeeded: [])
+        system.managers.forEach { $0.connect() }
```

Comment:
Is it weird that we have a stop but no start?

Reply:
Well, System.stop() can safely call disconnect() on every WalletManager, connected or not.  But, yes.  Let the App handle it?